src/main/primary_entry/script/common/ServiceURL.js

```diff
@@ -46,6 +46,7 @@ export default {
 
 
     TESLA_WEB_PAGE: 'http://www.tesla.com/findus/location/supercharger/',
+    TESLA_CN_PAGE: 'http://www.tesla.cn/findus/location/supercharger/',
```

Comment:
Maybe we should also change the two Tesla links to be pointed to https

Reply:
Good point, not sure how I missed that haha.